inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/gather_nd_ref.cl

```diff
@@ -0,0 +1,244 @@
+// Copyright (c) 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include "include/include_all.cl"
```

Comment:
You should use only those include files that you really need (for reducing of binary size).

Reply:
I included only "fetch.cl" instead of include_all.cl